src/twisted/python/failure.py

```diff
@@ -410,32 +430,33 @@ def _findFailure(cls):
         # difficult anyhow, so losing the Failure objects (and thus
         # the tracebacks) here when it is used is not that big a deal.
 
-        # handle raiseException-originated exceptions
+        # Handle raiseException-originated exceptions
         if lastFrame.f_code is cls.raiseException.__code__:
             return lastFrame.f_locals.get('self')
 
-        # handle throwExceptionIntoGenerator-originated exceptions
+        # Handle throwExceptionIntoGenerator-originated exceptions
         # this is tricky, and differs if the exception was caught
         # inside the generator, or above it:
 
-        # it is only really originating from
+        # It is only really originating from
         # throwExceptionIntoGenerator if the bottom of the traceback
         # is a yield.
         # Pyrex and Cython extensions create traceback frames
-        # with no co_code, but they can't yield so we know it's okay to just return here.
+        # with no co_code, but they can't yield so we know it's okay to
+        # just return here.
         if ((not lastFrame.f_code.co_code) or
             lastFrame.f_code.co_code[lastTb.tb_lasti] != cls._yieldOpcode):
```

Comment:
`lastFrame.f_code.co_code is an array of `bytes`.  On Python 2, `lastFrame.f_code.co_code[lastTb.tb_lasti]` returns a character, such as 'c'.  On Python 3, lastFrame.f_code.co_code[lastTb.tb_lasti]` returns an int, such as 99.